src/components/PaymentLog.vue

```diff
@@ -80,7 +80,7 @@
                   >{{ type.name }}</option>
                 </select>
               </div>
-              <div class="col form-group" v-if="renewalState && isAuthorized">
+              <div class="col form-group" v-if="isAltaraPay">
```

Comment:
Have we removed this two (renewalState && isAuthorized) computed values has well?